.changeset/forty-kids-exercise.md

```diff
@@ -0,0 +1,5 @@
+---
+"@chakra-ui/popover": major
```

Comment:
This should be `minor` since it's only adding backwards compatible functionality.